test/app_templates/config/initializers/shopify_app.rb

```diff
@@ -4,5 +4,6 @@
   config.secret = ENV['SHOPIFY_API_SECRET']
   config.scope = 'read_orders, read_products'
   config.embedded_app = true
-  config.session_repository = 'ShopifyApp::InMemorySessionStore'
+  config.shop_session_repository = 'ShopifyApp::InMemoryShopSessionStore'
+  config.user_session_repository = 'ShopifyApp::InMemoryUserSessionStore'
```

Comment:
Should we deafult to making them use a user_session or allow them to add it later? We could add another generator to add user session functionality then generate the User model and add this line?

Reply:
It is already like as you say, this line is to setup the InMemory storage, so that tests can run. There's a separate User generator that will set this to an ActiveRecord model

Reply:
ohhhhh thanks yeah I thought this was the template